### DISCUSSION TITLE
Does this support data sharding?

### DISCUSSION THREAD

**Discussion by hslee1064** [afs/rdf-delta#59] [Q&A] [answered]
Hello! I've been trying to make distributed triplestore system. But as far as I know, there is no open source project that supports RDF data sharding so far. So, I just wonder if this project support it or is going to support in the future.

Thanks.

**Comment by afs** [comment, marked as answer]
Hi there - no, RDF Delta does not support sharding data. 

It provides identical replicas of a database, not a partition of a database. So it can be used for increased query throughput; it does not give increased data scale.

**Reply by hslee1064** [reply]
Thank you! I think I need to find another way. I hope the project goes well!

**Reply by afs** [reply]
For storage at scale: [LDP](https://www.w3.org/TR/ldp/) may provide a way forward. e.g. [Trellis-LDP](https://github.com/trellis-ldp).
